Compilation Fails because of Log4j Dependency

Thibault reported this error when trying to compile Ict from source:

```
> Task :compileJava FAILED
warning: [options] bootstrap class path not set in conjunction with -source 1.7
~/iota/ict/src/main/java/org/iota/ict/api/RouteImpl.java:3: error: package org.apache.logging.log4j.core does not exist
import org.apache.logging.log4j.core.LogEvent;
                                    ^
~/iota/ict/src/main/java/org/iota/ict/utils/LogAppender.java:3: error: package org.apache.logging.log4j.core does not exist
import org.apache.logging.log4j.core.*;
```

We could trace it back to this commit: https://github.com/iotaledger/ict/commit/da2ba74244da6111bf5e8034736bf334ee74b914. Using the old version worked as intended. 
@ben-75 could you look into it?

--- a/build.gradle
+++ b/build.gradle
@@ -17,6 +17,7 @@ dependencies {
     compile "com.sparkjava:spark-core:2.7.2"
     compile 'org.json:json:20171018'
     compile "org.apache.logging.log4j:log4j-slf4j-impl:2.11.1"
+    compile "org.apache.logging.log4j:log4j-core:2.11.1"
 
     // used until pull request for iotaledger/iota.curl.java is merged
     compile 'com.github.mikrohash:iota.curl.java:master'

